apps/dashboard/src/components/primitives/control-input/variable-popover/hooks/use-suggested-filters.ts

```diff
@@ -0,0 +1,71 @@
+import { useMemo } from 'react';
+import { FILTERS } from '../constants';
+import { Filters, FilterWithParam } from '../types';
+
+type SuggestionGroup = {
+  label: string;
+  filters: Filters[];
+};
+
+export function useSuggestedFilters(variableName: string, currentFilters: FilterWithParam[]): SuggestionGroup[] {
```

Comment:
These provide filter suggestion based on common variable names, patterns of data